api/models/GroupJoinQuestionAnswer.js

```diff
@@ -1,16 +1,24 @@
 module.exports = bookshelf.Model.extend(Object.assign({
-  tableName: 'join_request_question_answers',
+  tableName: 'group_join_questions_answers',
```

Comment:
Been pondering about how we do join tables, since its a thing for responsibilities...

Should we defaulting to loading in user and group ids to these tables, to simplify lookups or SQL queries?

Reply:
thats what i did in this case, as you can see the migration. i think it depends on the use case

Reply:
in this case we need group and user because we wont always have a jon request anymore